net/respondd/src/respondd.c

```diff
@@ -649,10 +645,7 @@ int main(int argc, char **argv) {
 			break;
 
 		case 'd':
-			if (chdir(optarg)) {
```

Comment:
Yes, I just noticed that too.

Easiest fix would be to `chdir` to every directory that is given inside the `load_providers` function. That way, the first directory that has providers for a given request type has the chance to register the cache time.

Would that be okay?